src/core/Akka.Tests/Actor/InboxSpec.cs

```diff
@@ -122,11 +122,12 @@ public void Inbox_have_maximum_queue_size()
         [Fact]
         public void Inbox_have_a_default_and_custom_timeouts()
         {
-            Within(TimeSpan.FromSeconds(5), TimeSpan.FromSeconds(6), () =>
+            Within(TimeSpan.FromSeconds(4), TimeSpan.FromSeconds(6), () =>
             {
                 Intercept<TimeoutException>(() => _inbox.Receive());
                 return true;
             });
+
             Within(TimeSpan.FromSeconds(1), () =>
             {
                 Intercept<TimeoutException>(() => _inbox.Receive(TimeSpan.FromMilliseconds(100)));
```

Comment:
Looks like this part of the test is still subject to race conditions. Let's take a look at the Inbox code and see if anything's up there.